qm2/scf.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+import psi4
+
+np.set_printoptions(suppress=True, precision=4)
+
+
+def buildgeom:
```

Comment:
We may want to write the build_geom function to take an argument of a string. Let's ask AJ about this.